client/my-sites/customer-home/cards/tasks/index.jsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelFigure from 'components/action-panel/figure';
+import ActionPanelCta from 'components/action-panel/cta';
+import Gridicon from 'components/gridicon';
+import QueryPublicizeConnections from 'components/data/query-publicize-connections';
+import { savePreference } from 'state/preferences/actions';
+import { getPreference } from 'state/preferences/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { getConnectionsBySiteId } from 'state/sharing/publicize/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class RecommendedTask extends Component {
+	render() {
+		const { hideSocialTask, siteId, siteSlug, translate } = this.props;
+
+		if ( hideSocialTask ) {
+			return null;
+		}
+
+		let taskName;
+		let taskId;
+		let taskImage;
+		let taskDescription;
+		let taskMinutes;
+		let actionText;
+		let actionLink;
+		let shouldQueryPublicizeConnections;
+
+		if ( ! hideSocialTask ) {
+			taskName = translate( 'Connect social media accounts' );
+			taskId = 'SocialMedia';
+			taskImage = '/calypso/images/stats/tasks/social-links.svg';
+			taskDescription = translate(
+				'Integrate your site with social media to automatically post your content and drive content to your site.'
+			);
+			taskMinutes = '3';
+			actionText = translate( 'Connect accounts' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **37** times:
`translate( 'Connected accounts' )` *ES Score: 8*